.gitignore

```diff
@@ -26,3 +26,6 @@ password.txt
 # go dependancy
 /go.mod
 /go.sum
+
+# Slasher tmp db
+/slasherdata/slasher.db
```

Comment:
Why do we have to add this under gitIgnore? We dont do this for beacon DB

Reply:
removed